api/v2/Routes/students/progressquerystring/index.js

```diff
@@ -1,11 +1,11 @@
 import { Router } from 'express';
+import {getEmailFromAuth} from "../../../../lib/googleAuthHelper.mjs";
 import {getMaxPointsSoFar, getStudentScores} from "../../../../lib/redisHelper.mjs";
 import ProgressReportData from '../../../../assets/progressReport/CS10.json' assert {type: 'json'};
-import 'express-async-errors';
 
 const router = Router({ mergeParams: true });
 
-function mapTopicsToGrades(userGradeData) {
+/**function mapTopicsToGrades(userGradeData) {
```

Comment:
Ideally we should remove all unused code.  Leaving commented code in is generally bad practice.